backends/windows/kernelspace/proxier_events.go

```diff
@@ -147,7 +147,21 @@ func (Proxier *Proxier) onEndpointsMapChange(svcPortName *ServicePortName) {
 		// e := Proxier.endpointsMap[spn.NamespacedName]
 		endpoints := Proxier.endpointsMap[spn.NamespacedName]
 		for _, e := range *endpoints {
-			svcInfo.cleanupAllPolicies(e)
+			hns, _ := newHostNetworkService()
+			we := &windowsEndpoint{
+				ip:              e.IPs.First(),
+				port:            0, // TODO
+				isLocal:         e.Local,
+				macAddress:      "TODO",
+				hnsID:           "TODO",
```

Comment:
oh wait, we missed this somehow ? interesting - what is the upstream way to make these windowsEndpoints ? can you check

Reply:
yes I'll take a look !